internal/cli/root/atlas/builder_test.go

```diff
@@ -31,3 +39,91 @@ func TestBuilder(t *testing.T) {
 		[]string{},
 	)
 }
+
+func TestOutputOpts_printNewVersionAvailable(t *testing.T) {
+	tests := []struct {
+		currentVersion string
+		latestVersion  *version.ReleaseInformation
+		wantPrint      bool
+	}{
+		{
+			currentVersion: "v1.0.0",
```

Comment:
[q] Does the version info coming back also include the tag name prefix? (atlascli/v1.0.0)

Think that might break semver checks later in the code